src/viam/ur/module/ur_arm.cpp

```diff
@@ -84,7 +91,7 @@ T find_config_attribute(const ResourceConfig& cfg, const std::string& attribute)
     return *val;
 }
 
-enum class TrajectoryStatus { k_running = 1, k_cancelled = 2, k_stopped = 3 };
+enum class TrajectoryStatus : std::uint8_t { k_running = 1, k_cancelled = 2, k_stopped = 3 };
```

Comment:
we talked about ignoring it, but I like having an example of this in the code for future reference. would rather have this than more NOLINTS